slow_tests/test_barostat_slow.py

```diff
@@ -0,0 +1,111 @@
+# Test that a barostat'd waterbox has an equilibrated density very close to 1.0 kg/L
+# Analogous to https://github.com/openmm/openmm/blob/master/tests/TestMonteCarloBarostat.h#L204-L269
+# Expected runtime: ~ 10 minutes (10 replicates where each takes < 1 minute)
+
+import numpy as np
+from simtk import unit
+
+from testsystems.relative import hif2a_ligand_pair
+
+from md.builders import build_water_system
+from md.minimizer import minimize_host_4d
+
+from fe.free_energy import AbsoluteFreeEnergy
+
+from md.ensembles import PotentialEnergyModel, NPTEnsemble
+from md.barostat.moves import MonteCarloBarostat
+from md.barostat.utils import get_group_indices
+from md.states import CoordsVelBox
+from md.utils import simulate_npt_traj
+from md.thermostat.moves import UnadjustedLangevinMove
+from md.thermostat.utils import sample_velocities
+
+from timemachine.lib import LangevinIntegrator
+
+from functools import partial
+
+# simulation parameters
```

Comment:
Wrapping this in a unittest.TestCase seems cleaner than having this as a script

Reply:
Actually trying to run this, it took ~5 minute so perhaps best not as a test? Might be worth deciding on some convention between a 'test' and a 'benchmark'. As the slow tests still get run in CI

Reply:
I had mistakenly thought CI was running  `slow_tests/` less frequently than `tests/` (but I see that the actual behavior of `slow_tests/` is documented in [this PR description](https://github.com/proteneer/timemachine/pull/405) -- tests in this folder are run without cuda-memcheck so that they can be run at the same frequency as `tests/`, just faster). Happy to move this wherever is appropriate.

Do you think we should move this from `slow_tests/` into a new `benchmarks/` folder or something that doesn't get discovered by CI at all?

Another alternative might be to [mark](https://docs.pytest.org/en/6.2.x/mark.html) this and possibly other tests as slow, telling pytest to skip these most of the time, but periodically still run the slow tests through pytest.

Reply:
I say we leave this for now, as it is, this won't cause any issues. Probably warrants more discussion than a PR would allow